composer.json

```diff
@@ -11,16 +11,17 @@
         }
     ],
     "require": {
-        "php": "^5.4|7.*",
-        "puli/composer-plugin": "1.0.0-beta9"
+        "php": "^5.4|7.*"
     },
     "require-dev": {
         "php-http/httplug": "^1.0",
         "php-http/message-factory": "^1.0",
+        "puli/composer-plugin": "1.0.0-beta9",
         "phpspec/phpspec": "^2.4",
         "henrikbjorn/phpspec-code-coverage" : "^1.0"
     },
     "suggest": {
+        "puli/composer-plugin": "Allow to use Discovery",
```

Comment:
Yes, that description is not so good. The discovery page in the docs must also be improved as it is quite old.